geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -80,11 +80,14 @@
   public static final String MEMBER = "member";
   public static final String MEMBERS = "members";
   public static final String IFEXISTS = "if-exists";
+  public static final String IFNOTEXISTS = "if-not-exists";
```

Comment:
I like the unification of redundant variables, but you haven't replaced the two other uses of `if-not-exists`.

Reply:
@PurelyApplied Fixed it. Let me know if its good to go.